lunes_cms/cms/models/document.py

```diff
@@ -81,10 +81,11 @@ def converted(self):
         new_path = file_path[:-4] + "-conv.mp3"
         mp3_converted_file.export(new_path, format="mp3", bitrate="44.1k")
 
-        converted_audiofile = File(file=open(new_path, "rb"), name=Path(new_path))
-        converted_audiofile.name = Path(new_path).name
-        converted_audiofile.content_type = "audio/mpeg"
-        converted_audiofile.size = os.path.getsize(new_path)
+        with open(new_path, "rb") as file:
+            converted_audiofile = File(file=file, name=Path(new_path))
+            converted_audiofile.name = Path(new_path).name
+            converted_audiofile.content_type = "audio/mpeg"
+            converted_audiofile.size = os.path.getsize(new_path)
         os.remove(new_path)
         return converted_audiofile
```

Comment:
@timobrembeck 
This was what I've tried. I guess it's the cause of `seek of closed file` that `converted_audiofile` cannot reffer the file after `with` block is left.

Reply:
Ah ok interesting, I had hoped that `File()` would already read the file contents.
I don't have time to have a closer look at this problem, but I'll revisit this later.